src/pages/admin/TicketCards.vue

```diff
@@ -0,0 +1,450 @@
+<template>
+  <div class="">
+    <div
+      :class="this.ticketData.status === 'solved' ? 'opacity-25' : ''"
+      :style="expandState ? '' : ' overflow-y: scroll;height: 40rem'"
+      class="min-w-6/12 bg-white p-2 shadow rounded z-1"
+      style="width: 30rem;"
+    >
+      <div name="info" class="flex flex-col">
+        <div class="flex flex-row justify-between pb-2">
+          <div class="mr-2">
+            <div class="font-bold">{{ ticketData.name }}</div>
+            <div>{{ ticketData.email }}</div>
+            <div>{{ ticketData.phone }}</div>
+            <base-link text-variant="bodysm" class="px-2" target="_blank "
+              >{{ ticketData.organization.name }}
+            </base-link>
+            <base-button text="Tobi Events" variant="solid" class="m-2 p-1" />
+          </div>
+          <base-link
```

Comment:
IIRC, `base-link` has a boolean prop (`external` maybe?) that handles external links like this for you.